build/dependency

```diff
@@ -32,6 +32,13 @@ apply_patches() {
     fi
 }
 
+pkg_init() {
+    if [ -f Makefile ]; then
+        echo "- make distclean"
+        make distclean
+    fi
+}
```

Comment:
Add the `init` step, used by the `build/spec/mk` spec to cleanup the working dir